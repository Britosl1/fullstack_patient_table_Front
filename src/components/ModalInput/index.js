import React from 'react';
import { FormControl } from 'react-bootstrap'

function ModalInput({ searchValue, onChange, value }) {
  return (
    <FormControl
      value={searchValue}
      onChange={onChange}
      type="text"
      style={{ marginBottom: '10px' }} 
      placeholder={value}
      />
  )
}

export default ModalInput;