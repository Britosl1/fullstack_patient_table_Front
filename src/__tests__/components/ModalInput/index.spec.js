import React from 'react';
import renderer from 'react-test-renderer';
import ModalInput from '../../../components/ModalInput/index'

describe('Snapshot test', () => {
  it('renders correctly', () => {
    const nav = renderer
      .create(<ModalInput searchValue="Teste" onChange={jest.fn()} value="Value"/>)
      .toJSON();
    expect(nav).toMatchSnapshot();
  });

})