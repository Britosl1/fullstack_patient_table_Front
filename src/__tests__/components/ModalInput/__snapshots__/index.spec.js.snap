// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test renders correctly 1`] = `
<input
  className="form-control"
  onChange={[MockFunction]}
  placeholder="Value"
  style={
    Object {
      "marginBottom": "10px",
    }
  }
  type="text"
  value="Teste"
/>
`;
